ts/components/ForceScrollDownView.tsx

```diff
@@ -0,0 +1,204 @@
+        if (previousState && !thresholdCrossed) {
+          setButtonVisible(true);
+        }
+        return thresholdCrossed;
+      });
+    },
+    [threshold]
+  );
+
+  /**
+   * A side effect that calls the `onThresholdCrossed` callback whenever the value of `isThresholdCrossed` changes.
+   */
+  React.useEffect(() => {
+    onThresholdCrossed?.(isThresholdCrossed);
+  }, [onThresholdCrossed, isThresholdCrossed]);
+
+  /**
+   * A callback that is called whenever the size of the scrollable content changes. It updates the
+   * state with the new content height.
+   */
+  const handleContentSizeChange = React.useCallback(
+    (_contentWidth: number, contentHeight: number) => {
+      setContentHeight(contentHeight);
+    },
+    []
+  );
+
+  /**
+   * A callback that is called whenever the size of the scroll view changes. It updates the state
+   * with the new scroll view height.
+   */
+  const handleLayot = React.useCallback((event: LayoutChangeEvent) => {
+    setScrollViewHeight(event.nativeEvent.layout.height);
+  }, []);
+
+  /**
+   * A callback that is called when the "scroll to bottom" button is pressed. It scrolls the
+   * scroll view to the bottom and hides the button.
+   */
+  const handleScrollDownPress = React.useCallback(() => {
+    setButtonVisible(false);
+    scrollViewRef.current?.scrollToEnd();
+  }, [scrollViewRef]);
+
+  /**
+   * Whether or not the "scroll to bottom" button needs to be displayed. It is only displayed
+   * when the scrollable content cannot fit inside the scroll view and the button is enabled
+   * (`scrollEnabled` is `true`).
+   */
+  const needsScroll = React.useMemo(
+    () =>
+      scrollViewHeight != null &&
+      contentHeight != null &&
+      scrollViewHeight < contentHeight,
+    [scrollViewHeight, contentHeight]
+  );
+
+  /**
+   * Whether or not to render the "scroll to bottom" button. It is only rendered when the scroll view
+   * is enabled, needs to be scrolled, and the button is visible (`isButtonVisible` is `true`).
+   */
+  const shouldRenderScrollButton =
+    scrollEnabled && needsScroll && isButtonVisible;
+
+  /**
+   * The "scroll to bottom" button component. It is wrapped in a reanimated view and has enter and exit
+   * animations applied to it.
+   */
+  const scrollDownButton = (
+    <ScaleInOutAnimation
+      springConfig={IOSpringValues.button}
+      style={styles.scrollDownButton}
+      visible={shouldRenderScrollButton}
+    >
+      <IconButtonSolid
+        testID={"ScrollDownButton"}
+        accessibilityLabel="Scroll to bottom"
+        icon="arrowBottom"
+        onPress={handleScrollDownPress}
+      />
+    </ScaleInOutAnimation>
+  );
+
+  return (
+    <>
+      <ScrollView
+        testID={"ScrollView"}
+        ref={scrollViewRef}
+        scrollIndicatorInsets={{ right: 1 }}
+        scrollEnabled={scrollEnabled}
+        onScroll={handleScroll}
+        scrollEventThrottle={400}
+        style={style}
+        onLayout={handleLayot}
```

Comment:
```suggestion
        onLayout={handleLayout}
```
small typo, nothing exceptional

Reply:
Fixed in https://github.com/pagopa/io-app/pull/4541/commits/2fc36d2b748f0ee18110b401b86065a809354703